ethereum/core/src/main/java/tech/pegasys/pantheon/ethereum/mainnet/MainnetTransactionProcessor.java

```diff
@@ -231,6 +231,7 @@ public Result processTransaction(
               .blockchain(blockchain)
               .worldState(worldUpdater.updater())
               .initialGas(gasAvailable)
+              .transactionHash(transaction.hash())
```

Comment:
Why do we need the transaction hash for main-net?

Reply:
we don't